app-api/src/main/java/org/openmrs/eip/app/config/BeanAwareSpringLiquibase.java

```diff
@@ -0,0 +1,25 @@
+package org.openmrs.eip.app.config;
+
+import org.springframework.context.ApplicationContext;
+import org.springframework.core.io.ResourceLoader;
+
+import liquibase.integration.spring.SpringLiquibase;
+
+public class BeanAwareSpringLiquibase extends SpringLiquibase {
```

Comment:
What's the purpose of this?

Reply:
It was to be able to access context beans inside liquibaseChangeSet classes. But as you stated before, I replaced with the use of SyncContext.